android/assets/jsons/translations/template.properties

```diff
@@ -1190,6 +1190,7 @@ Water =
 water units = 
 wounded units = 
 Wounded = 
+Double movement in = 
```

Comment:
Do we actually use this string? Or do we only use "Double movement in [tileFilter]"? That should already be automatically added